1_The_Physical/Getting_Started_Emergent_Design/1_Red_Green_Refactor/1_2_Palindrome/src/index.spec.ts

```diff
@@ -1,4 +1,18 @@
-
+import { palindromeChecker } from "./index"
 describe('palindrome checker', () => {
 
+    it.each([
+        [true, 'mom'],
+        [true, 'Mom'],
+        [true, 'MoM'],
+        [false, 'Momx'],
+        [true, 'xMomx'],
+        [true, 'Was It A Rat I Saw'],
+        [true, 'Never Odd Or Even'],
+        [false, 'Never Odd Or Even1'],
+        [true, '1Never Odd Or Even1'],
+        [false, 'Never Even Or Odd'],
+    ])((`should return %s when given %s`), (expected,word) => {
```

Comment:
this is a simple recommendation I learned from Khalil, so I'd make my test spec more human-readable and avoid techy words, like `return false/true`, etc... so I'd prefer
` "it knows that 'mom' is a palindrome"` 
to 
`"it returns true for 'mom'”`

To do that in you code you can use array of elements instead of array of arrays in it.each(), you could separate success cases in a separate it.each() block, using `"it knows that %s is a palindrome"` for success cases it.each() block.

and `"it knows that %s is NOT a palindrome"` for failure cases it.each() block

so, it'd be something like
```ts
 it.each(['mom', 'Mom', 'MoM', 'xMomx', 'Was It A Rat I Saw'])
((`it knows that %s is a palindrome`), (input) => {
```

Reply:
Thanks for the comment. I will try in the next assignment!